Loop and Learn Edits

tmhastings opened the pull request:
This PR has been replaced by PR #52

tmhastings committed:
Update iphone.md

Correct formatting for admonitions

tmhastings committed:
Update pump.md

Correct link

tmhastings committed:
Update mainsettings.md

formatting correction

tmhastings committed:
Update trio.md

formatting corrections

tmhastings committed:
Update autosens-dynamic.md

change ICR to CR

tmhastings committed:
Update cgm.md

- update incorrect link
- grammar corrections

tmhastings committed:
Update Devices.md

- fix broken links
- grammar edits

tmhastings committed:
Update transition-qa.md

Clarify text

tmhastings committed:
Update transition-qa.md

- link correction
- grammar

tmhastings committed:
Update transition-qa.md

Add missing facebook link

tmhastings committed:
Update autosens-dynamic.md

correct link

tmhastings committed:
Update autotune.md

- reference "profile settings" earlier in document for clarity.
- minor grammar corrections

marionbarker commented:
Still in draft form so will not approve. Adding for your consideration. 

With LoopDocs, we use automatic translations via google translate. To make the automatic translations more robust, there are some language choices we make (more formal languages, fewer colloquial choices). One I noticed when looking at these diffs is the use of `carbs` instead of `carbohydrates`. I'm not sure how translations will be handled for Trio.

marionbarker commented on docs/EN/Configuration/Devices.md:
```diff
@@ -2,12 +2,12 @@
 
 ## Pump
-This section asks you to select your insulin delivery system. For more information on supported pumps, go to [Getting Started](../Getting-Started/pump.md). Omnipod Eros and older Medtronic devices don't support Bluetooth. These devices require a Rileylink, OrangeLink, or EmaLink to translate communication between your phone's Bluetooth signals and the pump's radio communication signals.
+This section asks you to select your insulin delivery system. For more information on supported pumps, go to [Getting Started](Getting-Started/pump.md). Omnipod Eros and older Medtronic devices don't support Bluetooth. These devices require a Rileylink, OrangeLink, or EmaLink to translate communication between your phone's Bluetooth signals and the pump's radio communication signals.
```
This link was correct before. Perhaps your reviewers were looking at an outdated version of the docs.
You must go up one level from Configuration and then select Getting-Started, so `(../Getting-Started/pump.md)` is correct.

tmhastings replied:
I thought so as well, but the link didn't work with the "../", but it did when I removed it. I will add it back and see if the error persists.

marionbarker commented on docs/EN/Configuration/Devices.md:
```diff
@@ -2,12 +2,12 @@
 
 ## CGM
-This section asks you to choose your glucose data source. Most options are self-explanatory. For more information on compatible CGMs, please go to [CGMs under Settings](../settings/devices/cgm.md).
+This section asks you to choose your glucose data source. Most options are self-explanatory. For more information on compatible CGMs, please visit [CGMs under Settings](settings/devices/cgm.md).
```
Ditto. `(../settings/devices/cgm.md)` looks correct to me.

marionbarker commented on docs/EN/Configuration/transition-qa.md:
```diff
@@ -30,7 +30,7 @@ meaning you can expect Trio to behave the same way AndroidAPS does generally.
 - If you have been successful with AndroidAPS but prefer the Apple ecosystem, you can easily transition to Trio. - If you've struggled to get stable BG with AndroidAPS, you may have the same issues with Trio.
   
-Take some time to read the docs and fine-tune your settings. To get help from other users, join [Discord](https://discord.gg/dbe5Twav8D) or [Facebook](https://www.facebook.com/groups/1351938092206709/). Once your settings are in order, consider switching to Trio.
+Take some time to read the docs and fine-tune your settings. To get help from other Trio users, join [Discord](https://discord.gg/dbe5Twav8D) or [Facebook](https://www.facebook.com/groups/1351938092206709/). Once your settings are in order, consider switching to Trio.
```
Is this the link we want in the docs. (Asking out of ignorance.)

marionbarker commented on docs/EN/Configuration/transition-qa.md:
```diff
@@ -136,21 +136,22 @@ Here's the recommendation:
 - Go through all the settings and tap the setting name to get an explanation of what each setting does.
-If you have any questions, feel free to reach out on [Discord](https://discord.gg/fCY5svg4) or Facebook. But please read the docs first :)
+
+If you have any questions, feel free to reach out on [Discord](https://discord.gg/fCY5svg4) or [Facebook](https://www.facebook.com/groups/diytrio). The best practice is to read the docs first, then ask the community any questions you have.
```
discord link?

marionbarker commented on docs/EN/settings/devices/cgm.md:
```diff
@@ -13,25 +13,25 @@ CGM selection includes the following:
 
 ## xDrip (xDrip4iOS)
-To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. You cannot use Shuggah or a version distributed by someone else's TestFlight. Please see the following for instructions on how to build xDrip4iOS yourself: [link](../../../operate/build.html#xdrip4ios-or-glucose-direct-as-cgm-source)
+To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. You cannot use Shuggah or a version distributed by someone else's TestFlight. Please see the following for instructions on how to build xDrip4iOS yourself: [link](operate/build.html#xdrip4ios-or-glucose-direct-as-cgm-source)
```
I think this link should be (../operate/build.md#xdrip4ios-or-glucose-direct-as-cgm-source) (or at least that's the format that would be used in mkdocs.

marionbarker commented on docs/EN/settings/devices/pump.md:
```diff
@@ -1,4 +1,4 @@
 # Pump
-This section allows you to select your insulin delivery system. For more information on supported pumps, see the following: [link](../../intro/requirements/pump.md). Older Omnipod and Medtronic devices require a Rileylink to translate communication between your phone's bluetooth signals and the pump's radiocommunication.
+This section allows you to select your insulin delivery system. For more information on supported pumps, see the following: [link](Getting-Started/pump.md). Older Omnipod and Medtronic devices require a Rileylink to translate communication between your phone's Bluetooth signals and the pump's radiocommunication.
```
I think this link should be {../../Getting-Started/pump.md)

tmhastings committed:
Update Overview.md

Correct links by adding "../"

tmhastings committed:
Update Overview.md

Remove Dropdown menus to allow for functioning links for CR, ISF, and Basal.

tmhastings committed:
Update Overview.md

Formatting correction

tmhastings committed:
Update Overview.md

Formatting correction

tmhastings committed:
Update Overview.md

Still fixing the same formatting issue

tmhastings committed:
Merge pull request #1 from nightscout/dev

Sync with Current Dev

tmhastings committed:
Merge branch 'LnL-edits' into tmhastings-LnL-Edits

tmhastings committed:
Merge pull request #2 from nightscout/tmhastings-LnL-Edits

Sync with Loop and Learn edits branch from nightscout

tmhastings committed:
Update Devices.md

Correct links

tmhastings committed:
Update Devices.md

Correct phrasing

tmhastings committed:
Update iphone.md

Capitalize "devices" in title

tmhastings committed:
Update iphone.md

- Capitalize Headings
- Update Formatting

tmhastings committed:
Update transition-qa.md

- Corrected formatting
- Update Facebook Link

tmhastings committed:
Transitioning from iAPS

- Add "Transitioning from iAPS" to the list of AID systems
- Link corrections
- Headings formatting

tmhastings committed:
Update transition-qa.md

Correct formatting for links

tmhastings committed:
Update transition-qa.md

Remove malfunctioning links

tmhastings committed:
Update transition-qa.md

Fix "Dynamic Settings" Header

tmhastings committed:
Update transition-qa.md

Convert Dynamic Settings warning under iAPS 2.3.3 heading to a caution admonition format

tmhastings committed:
Update transition-qa.md

Change "carb" to "carbohydrate"

tmhastings committed:
Update transition-qa.md

add line break before "Coming from iOS Loop" section

tmhastings committed:
Update transition-qa.md

formatting

tmhastings committed:
Update transition-qa.md

Loop section format correction

tmhastings committed:
Update transition-qa.md

tmhastings committed:
Update transition-qa.md

Update Introduction

tmhastings committed:
Update transition-qa.md

tmhastings committed:
Update transition-qa.md

tmhastings committed:
Update transition-qa.md

remove duplicate "chapter"

tmhastings committed:
Update autotune.md

tmhastings committed:
Update index.md

Move Autotune from Basic Settings to Advanced Settings

tmhastings committed:
Update Configure.md

Match reference to new menu name

tmhastings committed:
Update transition-qa.md

tmhastings committed:
Update Overview.md

tmhastings committed:
Update Overview.md

tmhastings committed:
Merge pull request #50 from tmhastings/LnL-edits

Update LnL Edits with personal Repo
